### PR TITLE
Wallet-click_CU-md8hjb

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-md8hjb Wallet click](https://app.clickup.com/t/md8hjb)

**Comment by lpboucher** [comment]
Task linked: [CU-md8hjb Wallet click](https://app.clickup.com/t/md8hjb)

**Comment by a-wilkes** [comment]
Same question as with [Cart-click](https://github.com/lpboucher/dak-coffee-wholesale/pull/45#issuecomment-860589252).

File: src/app/shared/components/wallet-modal/wallet-modal.component.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    templateUrl: "./wallet-modal.component.html",
+    styleUrls: ["./wallet-modal.component.scss"]
+})
+export class WalletModalComponent {
+    @ViewChild("modal") modal: ModalComponent | undefined = undefined;
+
+    constructor() { }
+
+    onOk(): void {
+        if (this.modal != undefined) {
```
Comment (lpboucher) [inline src/app/shared/components/wallet-modal/wallet-modal.component.ts]: minor change: prefer != null, the != checks for null and undefined

File: src/app/shared/components/wallet-modal/wallet-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Component, ViewChild } from "@angular/core";
+import { ModalComponent } from "@shared/components/modal/modal.component";
+
+@Component({
+    selector: "app-wallet-modal",
+    templateUrl: "./wallet-modal.component.html",
+    styleUrls: ["./wallet-modal.component.scss"]
+})
+export class WalletModalComponent {
+    @ViewChild("modal") modal: ModalComponent | undefined = undefined;
```
Comment (lpboucher) [inline src/app/shared/components/wallet-modal/wallet-modal.component.ts]: if a union type for undefined is passed there is no need to define an initial value

**Review by lpboucher — CHANGES_REQUESTED** [review]
This is approved. Will wait until the refactor is merged to merge this

**Pull request merged**